service/test/servicelayerinstance.cc

```diff
@@ -224,6 +224,13 @@ std::vector<chirp::Chirp> ServiceLayerInstance::Monitor(
   }
   return chirps_to_send;
 }
+std::vector<chirp::Chirp> ServiceLayerInstance::Stream(
+    const std::string &hashtag) {
+  chirp::StreamReply reply;
+
+  std::set<std::string> chirpsent;
```

Comment:
This seems a bit strange by itself here.

Reply:
Yes it doesn't do anything yet. I wanted to use it for the future so I put it there so I wouldn't forget.